### PR TITLE
PMM-7 remove reviewers from dependabot

### PR DESCRIPTION
**Pull request by BupycHuk**
Removing, because we fixed a problem with disappearing group

PMM-7

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

**Review by recharte — APPROVED** [review]

**Review by michal-kralik — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -4,11 +4,13 @@
...
 /managed/                               @percona/pmm-review-be
 /managed/services/dbaas                 @gen1us2k @percona-csalguero @percona/pmm-review-be
 /managed/services/management/dbaas      @gen1us2k @percona-csalguero @percona/pmm-review-be
-/update/                                @BupycHuk @atymchuk @talhabinrizwan
+/update/                                @BupycHuk @talhabinrizwan @percona/pmm-review-be
 /api-tests/                             @percona/pmm-review-be
-go.mod                                  @BupycHuk @artemgavrilov @percona/pmm-review-be
+**/go.mod                               @percona/pmm-review-dependency @percona/pmm-review-be
+**/Dockerfile                           @percona/pmm-review-dependency @percona/pmm-review-be
+/cli-tests/package.json                 @percona/pmm-review-dependency @yurkovich
```
Comment (artemgavrilov) [inline .github/CODEOWNERS:16]:
```suggestion
/cli-tests/package.json                 @percona/pmm-review-dependency @yurkovychv 
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -38,26 +32,15 @@ updates:
...
-
   - package-ecosystem: "npm"
     directory: "/cli-tests"
     schedule:
       interval: "weekly"
     reviewers:
       - "artemgavrilov"
       - "BupycHuk"
-      - "michal-kralik"
+      - "yurkovich"
```
Comment (artemgavrilov) [inline .github/dependabot.yml:45]: This `reviewers` section can be removed, right?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -4,11 +4,13 @@
...
 /managed/                               @percona/pmm-review-be
 /managed/services/dbaas                 @gen1us2k @percona-csalguero @percona/pmm-review-be
 /managed/services/management/dbaas      @gen1us2k @percona-csalguero @percona/pmm-review-be
-/update/                                @BupycHuk @atymchuk @talhabinrizwan
+/update/                                @BupycHuk @talhabinrizwan @percona/pmm-review-be
 /api-tests/                             @percona/pmm-review-be
-go.mod                                  @BupycHuk @artemgavrilov @percona/pmm-review-be
+**/go.mod                               @percona/pmm-review-dependency @percona/pmm-review-be
+**/Dockerfile                           @percona/pmm-review-dependency @percona/pmm-review-be
+/cli-tests/package.json                 @percona/pmm-review-dependency @yurkovychv
```
Comment (artemgavrilov) [inline .github/CODEOWNERS:16]: Missing newline

File: .github/dependabot.yml [hunk 4]
```diff
@@ -4,22 +4,17 @@ updates:
...
 
   - package-ecosystem: "gomod"
     directory: "/tools"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "gomod"
+    directory: "/api-tests/tools"
     schedule:
       interval: "weekly"
```
Comment (artemgavrilov) [inline .github/dependabot.yml:17]:
```suggestion
      interval: "daily"
```

Comment (artemgavrilov) [reply .github/dependabot.yml:17]: And for all other dependencies it makes sense to set `daily` as well

**Review by artemgavrilov — APPROVED** [review]

**Pull request merged**